### DISCUSSION TITLE
写了一个用于归档文件的手势，但是执行粘贴的时候总是粘贴回源文件夹，请帮忙看看哪里出了问题可以吗

### DISCUSSION THREAD

**Discussion by nx92532** [shuax/MouseInc.Settings#403] [General]
[

    ["SendKeys", "Ctrl+C"],

    ["Internal", "Delay", "200"],

    ["Execute", "\"H:\\资料库\\9月归档\""],

    ["Internal", "Delay", "500"],

    ["SendClick", "left"],

    ["Internal", "Delay", "500"],

    ["SendKeys", "Ctrl+V"]
]

一开始没有添加那个鼠标点击，我以为是窗口焦点没有放在新打开的窗口上。
加了点击后确认鼠标确实选中了新打开的窗口，但是执行C+V的时候依然粘贴到了源文件夹里。
不知道问题出在哪里，所以来请求大佬帮助了OTZ

**Comment by shuax** [comment]
因为SendKeys会自动激活触发窗口，你看看说明，你得加个参数不激活。

**Reply by nx92532** [reply]
找到了，需要加"NOACTIVATE"，谢谢

**Comment by 6Tom** [comment]
这个有点借机开眼了，还可以这么玩，想象力挺好，总上github闲逛就是好

我之想过一个类似的，跟你这个恰恰反过来，是我有固定的两个文件，时常需要把他们粘贴到新开的windows沙盒里

你这个其实借用上下文菜单要省事多了，因为你这个是复制，直接在“发送到”下面加上你的归档目录就行了，打开以下路径
```
%appdata%\Microsoft\Windows\SendTo\
```
按住alt把你的“H:\资料库\9月归档”这个文件夹拖进去，生成快捷方式，可以给快捷方式改名为“&9月归档”或“9月归档(&9)”，然后你就可以在选中的文件上直接右键，然后依次按“N”、“9”，就直接完事了

**Reply by nx92532** [reply]
好主意好主意，我直接在SENDTO里的快捷方式后加(&n),直接右键后双击n就进去了。又学到一招，这个用&指定快捷键的方法很实用。
